specifyweb/specify/tree_extras.py

```diff
@@ -498,8 +498,10 @@ def set_fullnames(table, treedefid, depth, reverse=False, null_only=False):
         parent_joins=parent_joins(table, depth),
         definition_joins=definition_joins(table, depth),
         null_only="and t0.fullname is null" if null_only else "",
+        node_number_range="and t0.nodenumber between {} and {}".format(node_number_range[0], node_number_range[1]) if not (node_number_range is None) else ''
     )
-    logger.debug('fullname update sql:\n%s', sql)
+
+    logger.warning('fullname update sql:\n%s', sql)
```

Comment:
Why was this changed to a warning and not kept as a debug? If this was used for testing purposes, please change this back to logger.debug() or a logger.info().
Of course, this can be kept as a warning if there is a valid reason for doing so

Reply:
that's because debug doesn't work for me so had to use warning. thanks for catching that!